posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -227,6 +231,24 @@ public class PostHog private constructor(
             getPreferences().setValue(DISTINCT_ID, value)
         }
 
+    private var isIdentified: Boolean = false
+        @JvmName("get-isIdentified")
+        get() {
+            synchronized(identifiedLock) {
+                if (!isIdentifiedLoaded) {
+                    println("hi")
```

Comment:
we can remove this

Reply:
Oh, my bad, I'm sorry I forgot to omit that.